rpcs3/Emu/SysCalls/Modules/cellVideoOut.cpp

```diff
@@ -111,7 +117,18 @@ s32 cellVideoOutGetDeviceInfo(u32 videoOut, u32 deviceIndex, vm::ptr<CellVideoOu
 {
 	cellSysutil.Warning("cellVideoOutGetDeviceInfo(videoOut=%d, deviceIndex=%d, info=*0x%x)", videoOut, deviceIndex, info);
 
-	if (deviceIndex) return CELL_VIDEO_OUT_ERROR_DEVICE_NOT_FOUND;
+	if (deviceIndex)
+	{
+		return CELL_VIDEO_OUT_ERROR_DEVICE_NOT_FOUND;
+	}
+
+	const auto avconfExt = fxm::get<avconfext_t>();
+	u32 gamma = 212; // Default value (1.0)
```

Comment:
This "gamma" parameter has different type, and may be completely unrelated (i.e. not affected by cellVideoOutSetGamma).